CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+Thanks for considering to contribute to the DLR Thermofluid Stream Library.
+
+Contributing can be done by either filing an issue or by contributing code.
+
+When filing an issue, please include all information necessary to reproduce,
```

Comment:
```suggestion
When filling an issue, please include all information necessary to reproduce,
```

Reply:
Hier kommt "filing" von "to file - einreichen", das sollte dann mit einem "l" passen